### PR TITLE
docs: remove mentions of #letsencrypt on Freenode.

### PR DESCRIPTION
**Pull request by cpu**
This PR removes two mentions of the `#letsencrypt` IRC channel on Freenode. This channel is being deprecated with the intention to consolidate on the community forum.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,8 +91,6 @@ Main Website: https://certbot.eff.org
 
 Let's Encrypt Website: https://letsencrypt.org
 
-IRC Channel: #letsencrypt on `Freenode`_
```
Comment (bmw) [inline README.rst]: Can you remove the now unused link for Freenode as well? That line is:
```
 .. _Freenode: https://webchat.freenode.net?channels=%23letsencrypt
```

Comment (cpu) [reply README.rst]: Sure! Will do. I didn't realize that's what the `_` notation was doing :+1:

**Comment by cpu** [comment]
>  If you prefer and/or can't get to it quickly though, I'll merge this as is and clean it up myself later.

@bmw I can push a revision after my next meeting if that works for you

**Comment by cpu** [comment]
@bmw This PR should be ready for another :mag: as of 23322a1. Thanks!

**Pull request merged**